src/components/DesktopApp/index.js

```diff
@@ -67,7 +52,7 @@ export default () => {
     const saveFileAs = () => saveFile({ saveAs: true })
     const exportToCSV = async () => {
       if (!file) return
-      let { cancelled, filePath } = await remote.dialog.showSaveDialog({
+      let { filePath } = await remote.dialog.showSaveDialog({
```

Comment:
presumably, we still want to check if the operation was cancelled- even though the code isn't there. I understand this was probably to fix a warning